hazelcast-client/src/main/java/com/hazelcast/client/connection/nio/AbstractClientSelectionHandler.java

```diff
@@ -45,7 +45,7 @@
     private final NonBlockingIOThread ioThread;
     @Probe
     private final int ioThreadId;
-    private volatile SelectionKey sk;
+    protected volatile SelectionKey sk;
```

Comment:
Hmm, CheckStyle should not be happy with the field order (`protected` after `private`), but I cannot spot a suppression for this class. Never mind, it's not a blocker anyway.